config/i18n-tasks.yml

```diff
@@ -94,6 +94,7 @@ ignore_unused:
 - 'errors.not_authorized'
 - 'experiments.*'
 - 'event_types.*'
+- 'headings.create_account_with*' # cannot find with interpolation
```

Comment:
The last time we allowed duplication in the 2FA views, it resulted in bugs, it cost Brendan extra time because he had to update multiple views instead of just one when updating the design, and it cost me many hours of refactoring.